x-pack/plugins/fleet/server/routes/package_policy/index.ts

```diff
@@ -106,7 +106,7 @@ export const registerRoutes = (router: FleetAuthzRouter) => {
       path: PACKAGE_POLICY_API_ROUTES.DRYRUN_PATTERN,
       validate: DryRunPackagePoliciesRequestSchema,
       fleetAuthz: {
-        integrations: { writeIntegrationPolicies: true },
+        integrations: { readIntegrationPolicies: true },
```

Comment:
I don't love this change, but seeing as how this API doesn't currently do any write operations, I guess it's ok. Ideally, we'd prevent the UI from calling this endpoint at all in the case where the user doesn't have write access to integration policies. @kpollich do you have any advice on where the best place to do this would be?

I'm ok with addressing this as tech debt or a follow up PR.

Reply:
I know it's not ideal. Currently the `POST dryrun` is allowed for users with Integrations `read` access, but it doesn't really write. These users are not allowed to save or update the policy though.

Reply:
👍 Can we file a tech debt issue for this?